hazelcast/src/main/java/com/hazelcast/config/DiscoveryStrategiesConfig.java

```diff
@@ -96,7 +96,7 @@ public boolean isEnabled() {
      * @return all enabled {@link DiscoveryStrategy} configurations
      */
     public Collection<DiscoveryStrategyConfig> getDiscoveryStrategyConfigs() {
-        return Collections.unmodifiableCollection(discoveryStrategyConfigs);
+        return discoveryStrategyConfigs;
```

Comment:
But this is another class whereas the other ones are lists of configs. This is a general difference but we probably just want to rename it to "DiscoveryConfig"?